tests/cross/src/test/scala/tests/pc/TypeDefinitionSuite.scala

```diff
@@ -0,0 +1,39 @@
+package tests.pc
+
+object TypeDefinitionSuite extends BaseTypeDefinitionSuite {
+
+  check("basic")(
+    """
+      |class Main(i: Int) {
+      |
+      |}
+      |
+      |object Main {
+      |  val ts@@t = new Main(2)
+      |}""".stripMargin,
+    "Main.scala [2:6 -> 2:9]"
+  )
+
+  check("method")(
```

Comment:
maybe rename to `constructor` ?
Then, the next case can be renamed to `method`

Reply:
Fixed.